xarray/tests/test_backends.py

```diff
@@ -1844,7 +1847,7 @@ def test_raise_on_forward_slashes_in_names(self) -> None:
     @requires_netCDF4
     def test_encoding_enum__no_fill_value(self):
         with create_tmp_file() as tmp_file:
-            cloud_type_dict = {"clear": 0, "cloudy": 1}
+            cloud_type_dict = {"clear": 0, "cloudy": 1, "missing": 255}
```

Comment:
Maybe, we can remove 
```suggestion
            cloud_type_dict = {"clear": 0, "cloudy": 1}
```